### ISSUE TITLE
Ability to copy full type definition without opening "Change Type" dialog

### ISSUE THREAD

**Issue by jonpalmisc** [Vector35/binaryninja-api#4472]
Often times I find myself copying struct definitions out of Binary Ninja into source code, etc. It would be handy to be able to copy a type's full definition without having to open the "Change Type" dialog.

"Copy" in the context menu (shown below) just copies the selected token.

![image](https://github.com/Vector35/binaryninja-api/assets/41548458/69869520-8b69-4018-9148-dbe1e3b5e239)

A "Copy Definition" option which would put the full type definition on the clipboard would be handy, such that if clicked for the enum pictured above, you would get the following on the clipboard:
```c
enum Fruit : uint32_t
{
    kFruitApple = 0x0,
    kFruitBanana = 0x1,
    kFruitOrange = 0x2,
    kFruitWatermelon = 0x3
};
```

In other words, whatever text would appear in the "Change Type" dialog if opened.

**Comment by xusheng6** [comment]
Related to: Copy #2151

**Comment by 0cyn** [comment]
Resolved in **dev builds >= 4.1.4983**
